data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```diff
@@ -67,6 +67,11 @@
       "description": "Optional. A UUID string that uniquely identifies the pre-auth token.",
       "type": "string",
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89aAbB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"
+    },
+    "corda.serial": {
+      "description": "Optional. The serial on the member's latest member info.",
+      "type": "string",
+      "maxLength": "100"
```

Comment:
Is there a reason we can't use an integer here, instead?

Also, it would be good to not have a limit (or have an extremely large one) to support scenarios with a large number of re-registrations.

Reply:
If I understand right, `"maxLength": "100"` means it can be a number up to 100 digits since it's a string length, right? Not that the limit of the number value is `100`. So I think it's already extremely high.

Reply:
Should be possible to use integer here if we want I think, we're doing that for the group policy file format version

Reply:
The group policy is not a map of string, string, it is actually a json file, hence it is a much easier case there.
We can add integer for the serial for sure (and define a min value as 0 and a max value), but then we need to convert back and forth, between the schema and the registration context.

Reply:
I see - ok, if that means 100 digits, we should be safe as-is.